packages/react-components/react-link/src/components/Link/useLinkStyles.styles.ts

```diff
@@ -86,11 +86,21 @@ const useStyles = makeStyles({
       color: tokens.colorNeutralForegroundDisabled,
     },
   },
+
+  inverted: {
+    color: tokens.colorBrandForegroundInverted,
+    ':hover': {
+      color: tokens.colorBrandForegroundInvertedHover,
+    },
+    ':active': {
+      color: tokens.colorBrandForegroundInvertedPressed,
+    },
+  },
```

Comment:
Do you need to handle `subtle` appearance on an inverted background as well?

Reply:
I'll leave it out until someone reports it, there are no designs that I've seen with subtle links in inverted background